src/snarky.d.ts

```diff
@@ -28,6 +28,8 @@ export { Test };
  * `Provable<T>` is the general circuit type interface. It describes how a type `T` is made up of field elements and auxiliary (non-field element) data.
  *
  * You will find this as the required input type in a few places in snarkyjs. One convenient way to create a `Provable<T>` is using `Struct`.
```

Comment:
```suggestion
 * You will find `Provable<T>` as the required input type in a few places in SnarkyJS. One convenient way to create a `Provable<T>` is using `Struct`.
```

Reply:
I am always on the lookout for adding a clear subject to replace "this" and "it"
see https://developers.google.com/style/pronouns#ambiguous-pronoun-references